content/en/announcements/otel-localized.md

```diff
@@ -0,0 +1,9 @@
+---
+title: OpenTelemetry website goes multilingual
```

Comment:
We can probably keep the exclamation mark here as well. It's kind of a big deal :)

Reply:
```suggestion
title: OpenTelemetry website goes multilingual!
```